tests/pipeline/Jenkinsfile.k8sSetup

```diff
@@ -6,8 +6,8 @@ pipeline {
           choice(name: 'node', choices: 'gce-reg\ngce-load', description: 'select env to run tests in ')
           choice(name: 'key', choices: 'master\nPR', description: 'Select master or a PR to run tests aganist - defaults to master')
           string(name: 'PR', defaultValue: '', description: 'PR number, required if key value is set to PR')
-          string(name: 'version', defaultValue: 'v0.5.0-111.ga4a1a98', description: 'Version to run tests against')
-          string(name: 'kube_version', defaultValue: 'v1.7.5', description: 'Version of kubernetes to  tests against')
+          string(name: 'version', defaultValue: 'v0.5.0-243.g9d8b216', description: 'Version to run tests against')
```

Comment:
is there a way to specify these settings in jenkins rather than committing in git? Then you could test different k8s or rook versions more easily.

Reply:
you can specify this settings in jenkins too, I put a default becaseu its easier when i am testing the jenkins script locally. i will leave it blank